source/localizable/partner_badge/index.html.md.erb

```diff
@@ -0,0 +1,113 @@
+---
+title: partner_badge
+---
+
+<%= partial 'partials/page_locales' %>
+
+# Partner Badge <%= edit_link %>
+
+"Partner Badge" acts as mark of trust and a quality indicator powered by <%= settings.site_name.capitalize %>.
+
+<%= partial 'partials/toc' %>
+
+## How It Looks
+
+### Floating
+
+##### Available Themes
+
+<div class="partner-badge-themes-preview floating-badge-themes-preview">
+  <%= image_tag "/assets/images/#{flavor}/partner_badge/floating_black.png" %>
+  <%= image_tag "/assets/images/#{flavor}/partner_badge/floating_orange.png" %>
+  <%= image_tag "/assets/images/#{flavor}/partner_badge/floating_white.png" %>
+</div>
+
+##### Preview
+
+<%= image_tag "/assets/images/#{flavor}/partner_badge/floating_preview.gif", class: 'partner-badge-preview' %>
+
+### Embedded
+
+##### Available Themes
+
+<div class="partner-badge-themes-preview embedded-badge-themes-preview">
+  <%= image_tag "/assets/images/#{flavor}/partner_badge/embedded_black.png" %>
+  <%= image_tag "/assets/images/#{flavor}/partner_badge/embedded_white.png" %>
+</div>
+
+##### Preview
+
+<%= image_tag "/assets/images/#{flavor}/partner_badge/embedded_preview.gif", class: 'partner-badge-preview' %>
+
+## How It Works
+
+The "<%= settings.site_name.capitalize %> Partner Badge" is displayed on your website, providing shop-related
+<%= settings.site_name.capitalize %> content upon click.
+
+- Acts as a **mark of trust** provided by <%= settings.site_name.capitalize %>, further increasing the shop's
+credibility.
+- Users will have **direct access** to shop-related <%= settings.site_name.capitalize %> content,
+such as rating, strong points, and user reviews.
+
+#### Partner Badge comes in two flavors:
+
+##### Floating
+
+"Partner Badge" is placed on a fixed position on the screen.
+
+##### Emebedded
+
+"Partner Badge" is placed on a static position of your choice and has the following dimensions: `width: 150px, height: 65px`
+
+## Integration
+
+"Partner Badge" requires to have [Analytics](/analytics/overview) integrated into your store properly.
+
+**Additionally**, *embedded* "Partner Badge" requires to put the following HTML element on your e-shop's pages:
+
+~~~ html
+<div id="sa-badge-embedded-plugin"></div>
+~~~
+
+
+### Technical Requirements
+
+Badge Display Type | [Analytics Integration](/analytics/overview) | Extra HTML element required
+------------------ | -------------------------------------------- | ---------------------------
+*Embedded*         |  ✔                                           | ✔ `<div id="sa-badge-embedded-plugin"></div>`
+*Floating*         |  ✔                                           | ✘
```

Comment:
I suggest using Yes/No instead of ✔/✘, mainly because the latter imply if certain requirements are already met.